{{cookiecutter.project_slug}}/.gitignore

```diff
@@ -5,7 +5,6 @@
 /config/*.json
 /config/*.yml
 /.setup_done
-/.wandb_sweeps
```

Comment:
Do [this](https://github.com/neuromation/cookiecutter-neuro-project/blob/ay/drop-wandb/%7B%7Bcookiecutter.project_slug%7D%7D/.gitignore#L15) guy is kept for user *wandb* usage in perspective?

Do we need [this](https://github.com/neuromation/cookiecutter-neuro-project/blob/ay/drop-wandb/.gitignore#L16) one?

Reply:
I agree that we should remove both.

Reply:
ok, got it